torch/nn/quantized/modules/conv.py

```diff
@@ -204,6 +203,12 @@ def get_qconv(cls, mod, activation_post_process, weight_post_process=None):
                     mod.stride, mod.padding, mod.dilation, mod.groups,
                     mod.bias is not None, mod.padding_mode)
         qconv.set_weight_bias(qweight, mod.bias)
+        try:
+            act_scale, act_zp = activation_post_process.calculate_qparams()
+        except Exception:
+            # for things like dynamic quantization, want to assign arbitrary s+z to not break serialization
```

Comment:
why do we call this from_float for dynamically quantized conv module? shouldn't we have a nn.quantized.dynamic.Conv2d?

Reply:
sure but the functions are exactly the same aside from dynamic not needing params